app/assets/javascripts/references.js

```diff
@@ -1,9 +1,18 @@
 $(document).ready(function() {
+  //Delete referene from a list
```

Comment:
Let's remove comments that explain _what_ is happening and not _why_ it's happening. What is happening _should_ be evident from the code itself.